bower.json

```diff
@@ -1,16 +1,16 @@
 {
-  "name": "aldryn-boilerplate-bootstrap3",
-  "private": true,
-  "dependencies": {
-    "bootstrap": "~3.3.4",
-    "bootstrap-sass-official": "~3.3.4",
-    "class.js": "~1.0.0",
-    "html5shiv": "~3.7.2",
-    "jquery": "~2.1.1",
-    "outdated-browser": "~1.1.0",
-    "qunit": "~1.17.0",
-    "respond": "~1.4.2",
-    "swfobject": "*",
-    "select2-bootstrap": "~1.3.0"
-  }
+    "name": "aldryn-boilerplate-bootstrap3",
```

Comment:
did you indent manually?) 
next `bower install --save` is gonna break it most likely